internetarchive/cli/ia_download.py

```diff
@@ -128,6 +132,18 @@ def main(argv, session: ArchiveSession) -> None:
         print(f'{exc}\n{printable_usage(__doc__)}', file=sys.stderr)
         sys.exit(1)
 
+    if args['--start-idx']:
+      start_idx = int(args['--start-idx'])-1
```

Comment:
Add spaces around all operators `int(args['--start-index']) - 1`. Also use four spaces for indentation as used everywhere else in the code ...
And if it should no be clear from the argument remark, also use start_index for the variable name instead of using idx.